### DISCUSSION TITLE
Aside from setting rebaseWhen to never, how to prevent Renovate bot from rebasing PRs automatically when main branch is updated?

### DISCUSSION THREAD

**Discussion by alvindizon** [renovatebot/renovate#19998] [Request Help] [answered]
### Type of discussion.

I just want to chat

### Tell us more.

Whenever our main branch is updated, any open PRs made by Renovate bot seem to be rebased automatically, based on my observation. I want to turn off this behavior since we want to conserve our minutes for CI/CD. I've researched and found that setting `rebaseWhen` to `never` should meet our requirement, but based on discussions I've seen [here](https://github.com/renovatebot/renovate/issues/13219), it seems that `rebaseWhen` causes problems in PR creation. Is there any alternative for `rebaseWhen`?

**Comment by viceice** [comment, marked as answer]
set to conflicted. then renovate only rebases automerge  and conflicted branches

**Reply by alvindizon** [reply]
Thanks @viceice !
